.github/workflows/buildkit.yml

```diff
@@ -73,6 +73,9 @@ jobs:
             disabledFeatures="${disabledFeatures},merge_diff"
           fi
           echo "BUILDKIT_TEST_DISABLE_FEATURES=${disabledFeatures}" >> $GITHUB_ENV
+      -
+        name: Expose GitHub Runtime
+        uses: crazy-max/ghaction-github-runtime@v2
```

Comment:
Could you add comments to explain why this is necessary?

Reply:
This will expose `ACTIONS_RUNTIME_TOKEN` and `ACTIONS_RUNTIME_URL` env vars otherwise gha cache integration test is skipped: https://github.com/moby/buildkit/blob/v0.12.2/client/client_test.go#L5407-L5411

@neersighted Can you add this comment? Don't want to jeopardize your branch :sweat_smile: